### PR TITLE
Enhance prow readme with enabling prow in new repo howto

### PR DESCRIPTION
**Pull request by kashifest**

### CODE REVIEW

**Comment by kashifest** [comment]
It seems markdown is not enabled here. I will do it in a separate PR.

File: prow/README.md [hunk 1]
```diff
@@ -184,3 +184,23 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
 Add the URL and token as below. Select **"Send me everything"**, and for Content ype: **application/json**.
 
 ![webhook](images/webhook.png)
+
+## Enabling metal3 prow for new org/repo
+
+metal3 prow is currently working with two github organizations(orgs): `metal3-io` and `Nordix`. For `Nordix` we have enabled prow only for two repositories, namely: metal3-dev-tools and metal3-clusterapi-docs. We don't forsee enabling metal3 prow for any other github org, however we might need to enable prow in other repositories in existing github orgs for example . In any case we should follow the steps below to enable prow:
```
Comment (furkatgofurov7) [inline prow/README.md:190]:
```suggestion
metal3 prow is currently working with two github organizations(orgs): `metal3-io` and `Nordix`. For `Nordix` we have enabled prow only for two repositories, namely: metal3-dev-tools and metal3-clusterapi-docs. We don't forsee enabling metal3 prow for any other github org, however we might need to enable prow in other repositories in existing github orgs for example. In any case we should follow the steps below to enable prow:
```

**Review by furkatgofurov7 — APPROVED** [review]

File: prow/README.md (hunk 1 above)
Comment (tuminoid) [inline prow/README.md:190]: - Capital M in all Metal3 names.
- Forsee -> foresee.
- github -> GitHub

File: prow/README.md [hunk 2]
```diff
@@ -184,3 +184,23 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
...
+
+1. Add/check `metal3-io-bot` user in the github org with `admin` access. Check the image 
+
+![](images/metal3-io-bot.png)
+
+2. Enable prow webhook as described in [GitHub webhook configuration](#github-webhook-configuration) section. For `metal3-io` the webhook is enabled in org level. For the two repositories in `Nordix` org we have enabled them on individual repositories. Keep in mind that the hmac token and hook URL is the same as described in [GitHub webhook configuration](#github-webhook-configuration). The webhook should look happy (green tik) as shown in the image below once you have configured correctly and communication has been established between github and prow hook.
```
Comment (tuminoid) [inline prow/README.md:196]: - hmac -> HMAC.
- token and hook URL is -> token and hook URL are
- tik -> tick
- configured correctly -> configured it correctly
- github -> GitHub

File: prow/README.md [hunk 3]
```diff
@@ -184,3 +184,23 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
...
+
+![](images/green_webhook.png)
+
+3. Check the general settings and branch protection settings of github repository and make sure the settings are correct. Take any existing repository which has prow enabled as example (i.e. `Nordix/metal3-dev-tools`).
+
+4. Add the repository entry and related configurations in the files `prow/config/config.yaml` and `prow/config/plugins.yaml` in `metal3-io/project-infra` repository. An example PR is [here](https://github.com/metal3-io/project-infra/pull/473/). 
+
+5. One small tweak might still be needed. We have experienced the default `merge_method` of prow which is `merge` didn't work for Nordix repos. The other two options for `merge_method` are: `rebase` and `squash`. We have enabled `rebase` for Nordix repos but kept `merge` for metal3-io org. An example is shown in this [PR](https://github.com/metal3-io/project-infra/pull/476/).
+
+6. At this point you should see the prow tests you have configured as presubmits for the repository, running on open PRs and tide is enabled and waiting for appropriate labels.
```
Comment (tuminoid) [inline prow/README.md:206]: `OWNERS` file is also needed before merging prow config. I don't see it mentioned here.

File: prow/README.md [hunk 4]
```diff
@@ -184,3 +184,23 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
...
+
+1. Add/check `metal3-io-bot` user in the github org with `admin` access. Check the image 
+
+![](images/metal3-io-bot.png)
+
+2. Enable prow webhook as described in [GitHub webhook configuration](#github-webhook-configuration) section. For `metal3-io` the webhook is enabled in org level. For the two repositories in `Nordix` org we have enabled them on individual repositories. Keep in mind that the hmac token and hook URL is the same as described in [GitHub webhook configuration](#github-webhook-configuration). The webhook should look happy (green tik) as shown in the image below once you have configured correctly and communication has been established between github and prow hook. 
+
+![](images/green_webhook.png)
+
+3. Check the general settings and branch protection settings of github repository and make sure the settings are correct. Take any existing repository which has prow enabled as example (i.e. `Nordix/metal3-dev-tools`).
```
Comment (tuminoid) [inline prow/README.md:200]: github -> GitHub

**Review by tuminoid — CHANGES_REQUESTED** [review]
Some nits.

File: prow/README.md (hunk 3 above)
Comment (kashifest) [reply prow/README.md:206]: missed it, good catch!

**Comment by kashifest** [comment]
/test-ubuntu-integration-main
@tuminoid PTAL again.

File: prow/README.md [hunk 5]
```diff
@@ -184,3 +184,25 @@ GitHub webhook needs to be configured with a payload URL pointing to the hook se
 Add the URL and token as below. Select **"Send me everything"**, and for Content ype: **application/json**.
 
 ![webhook](images/webhook.png)
+
+## Enabling metal3 prow for new org/repo
```
Comment (tuminoid) [inline prow/README.md:188]: One last lowercase metal3 here

**Review by tuminoid — APPROVED** [review]
/lgtm

**Pull request merged**